iron-lazy-page.html

```diff
@@ -0,0 +1,191 @@
+<!--
+Template element used in conjunction with `iron-lazy-pages`.
+
+@element iron-lazy-page
+-->
+<dom-module id="iron-lazy-page">
+</dom-module>
+<script>
+Polymer({
+  is: 'iron-lazy-page',
+
+  extends: 'template',
+
+  properties: {
+    /**
+     * The url corresponding to the content which should be lazy-loaded.
+     */
+    path: {
+      type: String,
+      value: ''
+    }
+  },
+
+  behaviors: [
+    Polymer.Templatizer
+  ],
+
+  /*
+    ##################
+    BEGIN DOM-IF LOGIC
+    ##################
+   */
+   _loadAndShow: function(cb, parent) {
+     var self = this;
+     var onImportFinished = function() {
+       cb(function() {
+         self._stamp(parent);
```

Comment:
`self._stamp(parent)` needs to be called prior to calling the callback function.
Otherwise the element won't be on the dom, making it so that the selectedAttribute and selectedClass don't get applied to the element. This is what's causing the tests to fail.